<!-- File: my_cv_camera_launch.launch -->

<launch>
    <node pkg="robot_nav" type="camera_detec.py" name="camera" output="screen" />
    <node pkg="robot_nav" type="lidar.py" name="robrot" output="screen" />
    
    <node name="rplidarNode" pkg="rplidar_ros" type="rplidarNode" output="screen">
        <param name="serial_port" type="string" value="/dev/usb_serial_cp210x" />
        <param name="serial_baudrate" type="int" value="256000" /><!--A3 -->
        <param name="frame_id" type="string" value="laser" />
        <param name="inverted" type="bool" value="false" />
        <param name="angle_compensate" type="bool" value="true" />
        <param name="scan_mode" type="string" value="Sensitivity" />
    </node>
    <node name="serial_node" pkg="rosserial_python" type="serial_node.py">
        <param name="port" type="string" value="/dev/usb_serial_hl340" />
        <param name="baud" type="int" value="115200" />
    </node>

</launch>